scripts/setupenv.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+APPDIR=/srv/www/todo_app
+
+if [[ $APPLICATIONTYPE == PRODUCTION ]]
+then
+    cd $APPDIR    
+    poetry run gunicorn --bind 0.0.0.0:5000 app:app
+elif [[ $APPLICATIONTYPE == DEVELOPMENT ]]
+then
+    poetry install && poetry add pytest && poetry add watchdog && poetry add pyyaml && poetry add argh
```

Comment:
I think you've since added the four additional packages here to your `pyproject.toml` so `poetry install` will already pick them all up